### DISCUSSION TITLE
Add reaction notifications

### DISCUSSION THREAD

**Discussion by pbodnar** [community/community#5911] [Other Feature Feedback, Questions, & Ideas]
This is basically a copy of isaacs/github#631 "E-mail notifications for reactions" from 2016, in order to request this highly-wanted feature again.

As suggested [here](https://github.com/isaacs/github/issues/631#issuecomment-297072154_), this is not necessarily just about email notifications, but about _any_ type of notification received when there's an emoji reaction to an issue comment or similar.

**Comment by rik** [comment]
Most reactions are sending good vibes to contributors so it's sad that they might never be viewed by them.

It might be nice to batch those notifications.

**Comment by benji-york** [comment]
I clicked the +1 reactions and then realized that I needed to add a comment to actually signal-boost this issue.  : (

**Reply by pbodnar** [reply]
thumb up ;)

**Reply by benji-york** [reply]
: )

**Comment by DaniMori** [comment]
We are using reactions in our workflows to notify when we "agree" with a comment or suggested change, so it would be really helpful if there were reaction notifications.

**Comment by vijayraghav-io** [comment]
Yes true how much ever reactions and up votes are done for the comments in this thread to push for this feature, i guess its not notifying anyone, unless a comment is dropped :) .  A much truly needed feature indeed! 
I should be visiting this page regularly to see the reaction for my comment :), It would be really nice if i get notified when there is a reaction.

**Comment by layerok** [comment]
Yeah, I also want to know if someone reacted to my comment to satisfy my ego.

**Reply by DaniMori** [reply]
That's legitimate enough 😝

**Comment by smheidrich** [comment]
This would be useful for open source projects that just started out and have no known users:

Often, authors will have already created a bunch of issues as TODOs for later, but working on them might be deferred until someone else actually needs the feature/fix in question. If such a potential user only leaves a :+1: instead of a comment (as is the etiquette), the author will probably never find out about it and hence not work on it further, because they're probably not spending a lot of time monitoring the issues, sorting them by reactions etc., which makes more sense to spend time on for established projects.

**Reply by DaniMori** [reply]
Couldn't agree more. Hope it's a well regarded feature for the near future! 🙂

**Comment by reslear** [comment]
that would be cool, like instagram.

**Comment by Liksium** [comment]
we need this 👍

**Comment by LouisMazel** [comment]
UP

**Comment by debtquity** [comment]
Yet another feature that is needed and has been sitting in the backlog graveyard of GH/MSFT.

I like @rik idea of batching the notifications on a configurable basis as well. I would like options for "daily", "weekly", or a custom schedule. 

 Particularly useful if an issue/subscribed thread/release goes viral with hundreds of reactions.

**Reply by Rudxain** [reply]
What about powers of some radix? That is, notify everytime a "milestone" is reached: 1, 2, 4, 8, ... This is the same as checking if the binary-logarithm has incremented its integer part

**Comment by coasting-nc** [comment]
GH/MSFT: Vibe code this already!
